### ISSUE TITLE
Mouse sensitivity slider

### ISSUE THREAD

**Issue by Swedneck** [minetest/minetest#6730]
While not critical to the enjoyment of the game, it's very annoying to have to go into the advanced settings in the main menu to change sensitivity.
There should be a sensitivity slider both in the main settings menu, and the ingame menu.

**Comment by SmallJoker** [comment]
For reference, more requested features for an in-game options menu: #6722
To prevent code duplication, we could use the same code from main menu in-game. Adding a slider to adjust the settings would then be a request for the settings menu in general.

**Comment by paramat** [comment]
We can't have everything in basic settings, and mouse sensitivity is not a basic setting, if you have mouse sensitivity correctly set on your computer it is rarely necessary to change it in the game :-1:

**Comment by Swedneck** [comment]
I understand that it might make things harder to maintain, but I just wanted to suggest something that would make the game more enjoyable. I really don't think having most settings available in the pause menu is that strange, seeing as most games have it that way.

IMO making things as convenient for the player as possible is very important.

**Comment by paramat** [comment]
Most games have large professional development teams, we have very limited dev time. The amount of work formspec settings (basic settings) takes is large.

The so-called 'advanced settings' is a well designed system that makes maintenance quick and easy, and isn't that difficult to use. It originally was THE settings system and replaced basic settings, but people complained (understandably because at the time the settings tree was badly designed and unintuitive) so it ended up being called 'advanced'.

So anything is basic settings has to be fundamental, and this is far from that, i've never used it and most will never need to. Having it in 2 places is even more unjustified. Other settings will have much more justification to be in basic settings, maybe you could suggest others? Also, it's higher priority to work on improving the advanced settings tree.

**Comment by ThomasMonroe314** [comment]
@paramat I agree with that, but the features being requested directly affect major gameplay (i.e. FOV), whereas alot of the settings in the "advanced settings" are not aimed at directly affecting how the player plays. I think having basic gameplay settings in the pause menu would be a nice feature to have.

**Comment by Calinou** [comment]
Being able to tweak mouse sensitivity is something I expect in any 3D application, so 👍.

**Comment by paramat** [comment]
>  I really don't think having most settings available in the pause menu is that strange,

We can't have most settings in the pause menu as formspecs because there are hundreds of them. They would have to be in a settings tree, which would end up as complex for the user as using advanced settings, but needing much more maintenance.

ThomasMonroe, this issue is requesting 'most settings' not just basic gameplay ones. I agree with you, the issue is, which are basic settings?

> Being able to tweak mouse sensitivity is something I expect 

Calinou, you can adjust it already.
It seems from what you write that on some systems the in-game sensitivity is inconsistent with the system setting? I assumed that the view direction movement would be equal to the movement of your computer pointer. This inconsistency is something i was not aware of which makes this more basic than i thought. However it is still something you set once so i can't see a priority for quick and easy access.

**Comment by Calinou** [comment]
> It seems from what you write that on some systems the in-game sensitivity is inconsistent with the system setting?

Indeed, especially with mouse acceleration settings differing from a system to another. (Minetest doesn't support raw input, which lets you eliminate mouse acceleration while keeping it enabled on the system, but that's another issue.)

**Comment by paramat** [comment]
Good point.

**Comment by ThomasMonroe314** [comment]
> I agree with you, the issue is, which are basic settings?

sound volume, FOV, and mouse sensitivity are what comes to my mind for basic gameplay settings, maybe shaders too, but idk

**Comment by paramat** [comment]
Any core dev support? (i'm neutral).
